cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -9,8 +9,8 @@ package api
 
 // KeyHandle represents a key with associated metadata.
 type KeyHandle struct {
-	Key   []byte `json:"key,omitempty"`   // Raw bytes
-	KeyID string `json:"keyID,omitempty"` //nolint: tagliatelle // False positive
+	Key   []byte `json:"key,omitempty"` // Raw bytes
```

Comment:
is it a public key?

Reply:
Updated to specify that it's public key bytes